arangod/Transaction/Manager.cpp

```diff
@@ -200,7 +200,8 @@ Manager::ManagedTrx::~ManagedTrx() {
                   opts);  // own state now
     TRI_ASSERT(trx.state()->status() == transaction::Status::RUNNING);
     TRI_ASSERT(trx.isMainTransaction());
-    trx.abort();
+    // TODO may this be ignored?
```

Comment:
I do not really see an option here, besides logging.
We are on the destructor of the transaction.